ocaml/doc/wire-protocol.md

```diff
@@ -170,7 +170,7 @@ to subequent RPC calls as an authentication token.
 A session can be terminated with the `session.logout` function:
 
 ```python
-   void  session.logout(session_id session)
+   void  session.logout(session ref)
```

Comment:
Oh, my comment was in reply to Marcello.

Regarding the name `session_id`, I don't think it's actually a names param in the API (it's positional), so it is just for the docs, and we could call it whatever we want. I think, however, that people casually take about this thing as a "session ID".